libjulia/backends/evm/EVMAssembly.cpp

```diff
@@ -145,6 +147,12 @@ void EVMAssembly::appendReturnsub(int _returns, int _stackDiffAfter)
 
 eth::LinkerObject EVMAssembly::finalize()
 {
+	size_t bytecodeSize = m_bytecode.size();
+	solAssert(uint64_t(bytecodeSize) < (uint64_t(1) << (8 * assemblySizeReferenceSize)), "Bytecode too big.");
+	for (auto const& ref: m_assemblySizePositions)
+		for (size_t i = 0; i < assemblySizeReferenceSize; i++)
```

Comment:
Can you write a helper for inserting big endian numbers? (same code is also used below)